src/main/java/org/osiam/storage/query/GroupFilterExpression.java

```diff
@@ -0,0 +1,63 @@
+package org.osiam.storage.query;
+
+import org.osiam.resources.scim.Group;
+import org.osiam.storage.entities.GroupEntity;
+
+import java.util.Locale;
+
+public class GroupFilterExpression extends FilterExpression<GroupEntity> {
+
+    private final Field field;
+
+    public GroupFilterExpression(String field, FilterConstraint constraint, String value) {
+        super(constraint, value);
+        this.field = new Field(field);
+    }
+
+    @Override
+    public Field getField() {
+        return field;
+    }
+
+    public static class Field implements FilterExpression.Field<GroupEntity> {
+        private String fieldName;
+        private GroupQueryField queryField;
+
+        public Field(String field) {
+
+            if (field.startsWith(Group.SCHEMA)) {
```

Comment:
I was unable to find a reference to case sensitivity of schemata, my gut feeling would be to be case sensitive but I don't really have a strong opinion on it. What shall we do?